src/main/java/org/quantumbadger/redreader/activities/OptionsMenuUtility.java

```diff
@@ -500,6 +502,29 @@ private static void addAllPostSorts(final AppCompatActivity activity, final Menu
 		addSort(activity, sortPostsTop, R.string.sort_posts_top_all, PostSort.TOP_ALL);
 	}
 
+	private static void addAllSubmittedSorts(final AppCompatActivity activity, final Menu menu, final boolean icon) {
```

Comment:
This is exactly the same as `addAllPostSorts` but omits `PostSort.RISING` from the sorts.  It could be cleaner to add a boolean argument to `addAllPostSorts` to drop the rising-sort?